test/parallel/test-eventtarget.js

```diff
@@ -50,7 +50,10 @@ ok(EventTarget);
   ev.preventDefault();
   strictEqual(ev.defaultPrevented, true);
 }
-
+{
+  const ev = new Event('foo');
+  deepStrictEqual(Object.keys(ev), ['isTrusted']);
```

Comment:
should this include a test that the value cannot be changed? I assume that's likely covered in the WPT tho?

Reply:
Indeed it is https://github.com/web-platform-tests/wpt/blob/master/dom/events/Event-isTrusted.any.js :]